openmaptiles/__init__.py

```diff
@@ -1 +1 @@
-__version__ = "5.2.0"
+__version__ = "test-5.0-pg12-3.0"
```

Comment:
why is this change here?

Reply:
Hi @nyurik, this PR is into `test-5.0-pg12-3.0` branch because it is the only one which has correct version of PostgreSQL and PostGIS. I didn't make the PR into master because it didn't make sense to me add it there when the script requires PostgreSQL12+. This `__version__` is used as a tag for a Docker image during the build  and I though the tag should correspond to branch name. However if that is not correct please let me know what version should be there and whether I chose a right branch. Thanks!

Reply:
@lazaa32 i think we shouldn't change it here because ST_AsMVT works in Postgres v9.6 just as well (technically it was first implemented in PostGIS 2.4, and we have been using it ever since). If you want to use PG12, just set the appropriate var in the .env file.